NOTICE

```diff
@@ -1,8 +1,15 @@
-Copyright 2018 RDK Management
-Licensed under the Apache License, Version 2.0 
-
+This component contains software that is Copyright (c) 2018 RDK Management.
```

Comment:
Hi Punam: Please will you add some blank lines because this file needs to be very clear.  (This is happening because of cut/paste across different tools.)  Blank lines after lines 3, 7, 9, 11, 13.)
If easier, here is exactly what I want, assuming github doesn't eat the lines:

This component contains software that is Copyright (c) 2018 RDK Management.
The component is licensed to you under the Apache License, Version 2.0 (the "License").
You may not use the component except in compliance with the License.

The component may include material which is licensed under other licenses / copyrights as
listed below. Your use of this material within the component is also subject to the terms and
conditions of these licenses. The LICENSE file contains the text of all the licenses which apply
within this component.

Copyright (C) 2012 Raphael Kubo da Costa <rakuco@webkit.org>
Licensed under the BSD-2 license

Based on pxCore, Copyright 2015-2018 John Robinson
Licensed under the Apache License, Version 2.0

Copyright 2022 Sky UK
Licensed under the Apache License, Version 2.0

Copyright (C) 2020 RDK Management
Licensed under the BSD-2 license